test/integration/privacyGroupManagement.test.js

```diff
@@ -0,0 +1,32 @@
+const test = require("tape");
+
+const createGroup = require("../../example/privacyGroupManagement/createPrivacyGroup");
+const findGroup = require("../../example/privacyGroupManagement/findPrivacyGroup");
+const deleteGroup = require("../../example/privacyGroupManagement/deletePrivacyGroup");
+
+test("[MultiNodeExample]: Can manage privacy groups", t => {
+  let RESULT;
+  t.test("can create privacy group", async st => {
+    RESULT = await createGroup();
+
+    st.end();
+  });
+
+  t.test("can find privacy group", async st => {
+    const result1 = await findGroup();
+
+    st.equal(RESULT, result1);
+
+    st.end();d
+  });
+
+  t.test("can delete privacy group", async st => {
```

Comment:
We should have a bunch more testing around all the negative test cases.

Reply:
Have added a few more tests now. Though they dont all pass, waiting for all PRs in Pantheon to be merged.